src/main/java/org/apache/commons/csv/CSVRecord.java

```diff
@@ -48,6 +48,11 @@ public final class CSVRecord implements Serializable, Iterable<String> {
      */
     private final long characterPosition;
 
+    /**
+     * The start byte of this record as a character byte in the source stream.
+     */
+    private final long characterByte;
```

Comment:
This name is too confusing IMO, please rename to `bytePosition` to mirror the existing `characterPosition`.